pkg/datapath/prefilter/prefilter.go

```diff
@@ -250,15 +209,9 @@ func (p *PreFilter) init() (*PreFilter, error) {
 
 // NewPreFilter returns prefilter handle
 func NewPreFilter() (*PreFilter, error) {
-	haveLPM := probe.HaveFullLPM()
```

Comment:
There a a few other instances of `probe.HaveFullLPM` in `daemon/cmd`. I think these can be removed as well along with `probe.HaveFullLPM` itself.